cruise.umple/test/cruise/umple/compiler/UmpleParserTest.java

```diff
@@ -2560,6 +2560,70 @@ public void methodNameConflictUsingDefaulted()
     assertFailedParse("045_phpDefaultedNameConflict.ump", 48);
     assertFailedParse("045_rubyDefaultedNameConflict.ump", 48);
   }
+  
+  @Test
+  public void autoGeneratedAPIMethods()
```

Comment:
Is there a way to only have to call assertSimpleParse once if I separate them all into separate tests? Or do I have to call it now for each test?